src/server.rs

```diff
@@ -1298,7 +1297,7 @@ where
             Ok::<_, Error>(())
         };
 
-        self.pool.spawn(Box::pin(async move { task.await; } )).unwrap();
+        self.pool.spawn(Box::pin(async move { task.await.unwrap(); } )).unwrap();
```

Comment:
Since we just spawn a task, on the pool, it's not good to bring down the whole application if one task fails, rather `warn!` and continue operation.
(If spawning a new task on the other hand fails, that's a rather big deal, but should be `.except`)